src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -131,24 +131,20 @@ public override int GetHashCode()
             }
         }
 
-        private struct ReadOnlyDataBlobKey : IEquatable<ReadOnlyDataBlobKey>
+        private struct SettableReadOnlyDataBlobKey : IEquatable<SettableReadOnlyDataBlobKey>
         {
             public readonly Utf8String Name;
-            public readonly byte[] Data;
-            public readonly int Alignment;
 
-            public ReadOnlyDataBlobKey(Utf8String name, byte[] data, int alignment)
+            public SettableReadOnlyDataBlobKey(Utf8String name)
```

Comment:
Sigh, sorry to pester one more time - since the alignment parameter is now gone from here, this struct can be deleted too. We can just use the `Utf8String` as the key to the `NodeCache` instead.

Reply:
No problem. Thank you so much!